### ISSUE TITLE
there are often errors in ``instance-management"

### ISSUE THREAD

**Issue by liwh1111** [sitewhere/sitewhere#938]
Hello, there are often errors in ``instance-management", have you encountered it during testing?

![image](https://user-images.githubusercontent.com/63097736/107311650-b1ddb580-6ac9-11eb-8be7-42206c121cf1.png)

![image](https://user-images.githubusercontent.com/63097736/107311745-d76abf00-6ac9-11eb-9536-cf16b2f0af40.png)

I feel that the visit is always inaccessible, I don’t know why, I clicked again to visit again

**Comment by derekadams** [comment]
This appears to be a bug in the device management microservice when attempting to list device commands. Error show up in the instance management microservice because it is the hub for REST service calls. We will look into fixing this for the next CE release.

**Comment by liwh1111** [comment]
Has this problem been solved in version 3.0.4

**Comment by derekadams** [comment]
Yes, this should be working in version `3.0.4`. We fixed a number of issues in the device management APIs, but please report if you find any other issues.
